var/spack/repos/builtin/packages/abinit/package.py

```diff
@@ -209,27 +215,22 @@ def configure_args(self):
                     oapp('--with-fft-libs=-L{0} {1}'.format(
                         spec['fftw-api'].prefix.lib, fftlibs))
         else:
-            if '+openmp' in spec:
-                fftflavor, fftlibs = 'fftw3-threads', '-lfftw3_omp -lfftw3 -lfftw3f'
+            if '^mkl' in spec:
+                fftflavor = 'dfti'
+                options.extend([
+                    'FFT_CPPFLAGS={0}'.format(spec['fftw-api'].headers.cpp_flags),
+                    'FFT_LIBs={0}'.format(spec['fftw-api'].libs.ld_flags),
+                ])
+            elif '^fftw' in spec:
+                if '+openmp' in spec:
+                    fftflavor, fftlibs = 'fftw3-threads', '-lfftw3_omp -lfftw3 -lfftw3f'
+                else:
+                    fftflavor, fftlibs = 'fftw3', '-lfftw3 -lfftw3f'
```

Comment:
A similar change will be needed for the `@:8` case.

Reply:
done